modules/katello_host_collection.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2019, Maxim Burgerhout <maxim@wzzrd.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: katello_host_collection
+short_description: Create and Manage Katello host collections
+description:
+    - Create and Manage Katello host collections
+author:
+    - "Maxim Burgerhout (@wzzrd)"
+requirements:
+    - "nailgun >= 0.32.0"
+    - "python >= 2.6"
+    - "ansible >= 2.3"
+options:
+  server_url:
+    description:
+      - URL of Foreman server
+    required: true
+  username:
+    description:
+      - Username on Foreman server
+    required: true
+  password:
+    description:
+      - Password for user accessing Foreman server
+    required: true
+  verify_ssl:
+    description:
+      - Verify SSL of the Foreman server
+    default: true
+    type: bool
+  description:
+    description:
+      - Description of the Katello host collection
+    required: false
+  organization:
+    description:
+      - Organization that the Product is in
+    required: true
+  name:
+    description:
+      - Name of the Katello host collection
+    required: true
+  state:
+    description:
+      - State of the host collection
+    default: present
+    choices:
+      - present
+      - absent
+      - present_with_defaults
+'''
+
+EXAMPLES = '''
+- name: "Create Foo host collection"
+  katello_host_collection:
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    name: "Foo"
+    description: "Foo host collection for Foo servers"
+    organization: "My Cool new Organization"
+    state: present
+'''
+
+RETURN = '''# '''
```

Comment:
Here should be an additional space.

See
https://docs.ansible.com/ansible/2.3/dev_guide/developing_modules_documenting.html#return-block

Reply:
Took me a bit to figure out what you meant there. Other modules in the repo I compared mine with do not have that space either :)

Reply:
Yes, i think we should get it right someday. It does not seem to be a real problem though. The linter does not complain. I just looks ugly.